shuffle/move/examples/main/Move.toml

```diff
@@ -3,7 +3,7 @@ name = "Message"
 version = "0.0.0"
 
 [addresses]
-Sender = "0x24163afcc6e33b0a9473852e18327fa9" # TODO: Hardcoded to sender addr
+Sender = "_"
```

Comment:
🎉 

This will affect one's ability to use `move package build/test` directly against the package, which may affect the tutorial. cc @emmazzz @tzakian